burn-import/src/onnx/to_burn.rs

```diff
@@ -563,6 +567,21 @@ impl ONNXGraph {
         MaxPool2dNode::new(name, input, output, config)
     }
 
+    fn conv_transpose2d_conversion(node: Node) -> ConvTranspose2dNode {
+        let input = node.inputs.get(0).unwrap().to_tensor_type();
+        let weight = node
```

Comment:
TODO: process not only constant weights

Reply:
This should be similar to Conv2d: https://github.com/burn-rs/burn/blob/main/burn-import/src/onnx/to_burn.rs#L560